route/build/src/standalone/get_basin_runoff.f90

```diff
@@ -23,9 +23,11 @@ SUBROUTINE get_hru_runoff(ierr, message)
   USE public_var,           ONLY: vname_qsim          ! varibale runoff in netCDF file
   USE public_var,           ONLY: vname_evapo         ! varibale actual evaporation in netCDF file
   USE public_var,           ONLY: vname_precip        ! varibale precipitation in netCDF file
+  USE public_var,           ONLY: vname_cc            ! varibale name of a constituent [g] in netCDF file
```

Comment:
variable is misspelled here. And again maybe you also mention solute in the comment.

Reply:
`vname_cc` is changed to `vname_solute`, `constituent` to `solute`